services/worker-manager/README.md

```diff
@@ -2,6 +2,11 @@
 
 The worker manager service manages workers, including interacting with cloud services to create new workers on demand.
 
+It currently includes providers for:
```

Comment:
I'd prefer to see some more documentation explaining the overall architecture of the thing